### DISCUSSION TITLE
[deepin 20.7]Intel AC 9560网卡休眠启动后无线网卡无法使用，rfkill hard lock

### DISCUSSION THREAD

**Discussion by Skywol** [linuxdeepin/developer-center#3467] [Troubleshooting | 故障排查]
问题：笔记本在休眠唤醒后，wifi无法使用，无线网卡rfkill hard lock，蓝牙消失。
说明：笔记本不存在物理wifi开关键，但有FN+F10的飞行模式键
配置：
![截图_deepin-devicemanager_20221016130159](https://user-images.githubusercontent.com/9556429/196019190-49c7c499-fd71-4813-861a-9c74639b8dfc.png)
复现步骤：
![录屏_dde-desktop_20221016125733](https://user-images.githubusercontent.com/9556429/196019198-9614c8a6-ec0b-4c7f-acc1-4e97d7c3ec07.gif)
已尝试过的解决方案：
1.blacklist wmi 无效
2.增大swap，试过4G、16G、32G无效
日志：
[系统全部日志.zip](https://github.com/linuxdeepin/developer-center/files/9793645/default.zip)

**Comment by BLumia** [comment]
cc @Zeno-sole
